.github/workflows/directx-unit-test.yml

```diff
@@ -0,0 +1,58 @@
+name: DirectX unit test
```

Comment:
Why related to this PR?

Reply:
not related to this PR, just I have no confidence on this pipeline and cannot test it locally, so temporarily leverage this PR to test it, will create another PR for it when it works.

Reply:
Sure, thanks. I have approved this PR. Please make sure these code has been removed.